### DISCUSSION TITLE
Building AirSim

### DISCUSSION THREAD

**Discussion by Efrit16** [microsoft/AirSim#3454] [Support Q&A]
Hello everyone, I am having some trouble building AirSim on my computer. I am following the steps from https://microsoft.github.io/AirSim/unreal_custenv/ ( Windows) However, in step 4, when I create a C++ class, this message pops up : " Successfully added class 'MyClass', however you must recompile the 'LandscapeMountains' module before it will appear in the Content Browser. Failed to generate project files.

Would you like to open the Output Log to see more details? "
.ccp and .h were created, but Landscapes.sln was not.
It would be very appreciated if I get help as I want to contribute to this project a lot!

An alternative would also be welcome, thanks!

**Comment by xxEoD2242** [comment]
@Efrit16 

This generally has to do with Windows not correctly installing the Unreal Engine tools. What you can try is to open your project using the Epic Games/Unreal Engine launcher. When you open a project, it will automatically trigger a recompile of the code.

Take a look at this forum as well. It could be an issue with Windows not seeing Visual Studio [Unreal Doesn't See Visual Studio](https://forums.unrealengine.com/unreal-engine/unreal-studio/1520136-solved-unreal-don-t-see-visual-studio-2017)

**Reply by Efrit16** [reply]
@xxEoD2242 
Thanks for the help, seems to work correctly now. It turned out my .NET framework was not up to date
